### ISSUE TITLE
FIM System tests: Create first scenario to integrate all actions together

### ISSUE THREAD

**Issue by rshad** [wazuh/wazuh-qa#480]
#### Objective

Implement Ansible tasks to run each pair (action,tests) and a global Ansible playbook to launch the pairs (action,test) in a determined order `for testing purposes`.

#### Tasks

- [x] Create required variables for all actions and playbooks (@jm404 )

- [x] Create ossec.conf and defaults configuration for the scenario (@DFolchA )

- [x] Create playbook `launch_scenario.yml` which has the following subtasks
  - [x] Prepare tasks @rshad (create folders and load the ossec.conf file)
  - [x] Create files @manuasir @rshad @xr09 
  - [x] Modify files @jm404 @xr09 
  - [x] Delete files @jm404 @xr09 

- [x] Create tasks for verify JSON and ES tests with updated variables (@DFolchA)

- [x] Test whole scenario together

**Comment by rshad** [comment]
Working branch: [feature-480-ansible-tasks-actions-tests](https://github.com/wazuh/wazuh-qa/tree/feature-480-ansible-tasks-actions-tests).

**Comment by xr09** [comment]
#### Contributions

- Updated parameters for generate and modify scripts. (following the defined format)
- Tuned tasks for executing file creation, change and deletion on Linux and Windows
- Improved file creation algorithm (less ram usage, making every file truly unique)

**Comment by DFolchA** [comment]
Hello team,

For this issue I have contributed the following items:

- Continue working on  `launch_scenario.yml`  with @jm404 
- Create `handlers` for the scenarios.
- Create tasks to launch the alert verification scripts.
- Fixed some mistakes with `become` and windows tasks
- Created the `ossec.conf` files for this scenario.

I have also tested the scenario in my local environment.

Regards,
Daniel Folch
